### DISCUSSION TITLE
This user account is blocked. Help <3

### DISCUSSION THREAD

**Discussion by bilalgursen** [vercel/community#4490] [Account Issue] [answered]
## GitHub Login Issue with Vercel

Hello Vercel Support Team,

My name is Bilal Gürşen, and I am encountering an issue while trying to log in to Vercel using my GitHub account. When I attempt to log in with GitHub, I receive an error message stating, "This user account is blocked," and I am in need of your assistance to resolve this matter.

I'm not entirely sure why my account has been blocked or how to address it. Could you please provide me with insights on how to overcome this access restriction to Vercel?

<img width="1710" alt="Ekran Resmi 2023-10-16 17 41 56" src="https://github.com/vercel/community/assets/107859927/ea3ab5e1-fda8-4707-b31e-c7bad4ff8d2e">
<img width="1710" alt="Ekran Resmi 2023-10-16 17 42 16" src="https://github.com/vercel/community/assets/107859927/18a06dd3-5a4d-4c35-b492-ae62cb14414c">

I have not engaged in any security breaches or misuse that would warrant my account's suspension. My sole intention is to deploy my Next.js 13 project on Vercel, and I require a solution to this issue.

Could you kindly assist me in resolving this matter? I would greatly appreciate your guidance on the necessary steps to take.

Thank you.

**Comment by swarnava** [comment, marked as answer]
You can email us at registration@vercel.com
